lib/railway_ipc/consumer_group/consumer_supervisor.ex

```diff
@@ -0,0 +1,21 @@
+defmodule RailwayIpc.ConsumerGroup.ConsumerSupervisor do
```

Comment:
What's the difference between the ConsumerSupervisor and  the Supervisor?

Reply:
![image](https://user-images.githubusercontent.com/325485/92967580-85a10880-f447-11ea-8a6d-956be886fb47.png)
We're building something like that. The Supervisor is the parent for both the consumer supervisor and the monitor. This lets you restart the tree from the parent if needed.

Reply:
The consumer supervisor is just a supervisor that starts _your_ consumers.